### PR TITLE
Create delivery employee table 

### PR DESCRIPTION
**Pull request by Criofann**
Adding the first set of scripts to the main directory so the rest of the team can work on it.
This addition allows for the creation of the delivery employee table as well as some example data that is being entered into the table for testing purposes.

Proposed solution for point 2 of the product backlog "As a member of the HR team I want to be able to create a new delivery employee. I should be able to store a name, salary, bank account number and national insurance number"

### CODE REVIEW

**Comment by lullabelle** [comment]
I would maybe call this script Delivery_Employee_table or have naming convention such as CreateDeliveryEmployee.sql because I think you might also have to create sales employee and might be difficult to distinguish between the two.

Also dont be afraid to indent your sql! Easier to read that way!

Are you missing a commission rate?

**Review by lullabelle — CHANGES_REQUESTED** [review]
I would maybe call this script Delivery_Employee_table or have naming convention such as CreateDeliveryEmployee.sql because I think you might also have to create sales employee and might be difficult to distinguish between the two.

Also dont be afraid to indent your sql! Easier to read that way!

**Review by lullabelle — APPROVED** [review]
Good to go!

File: Delivery_Employee_table.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+INSERT INTO Delivery_Employee (FirstName, LastName, Salary, BankAccount, NationalInsurance )
+  VALUES ('Joseph', 'Magee', 19000 ,'87212667578643', 'AD913336R'),
+    ('Samuel','Buyan',19000,'21785671234598','AR650465N'),('John', 'Smith', 100000, '34422315894589', 'HG124809K'),
+    ('Joshua','Kelso',900000,'21932721875632','KF242148H'),('Harry','Potter' ,12000, '81760910454545','YT877799U'),
+    ('Mario','Mario',50000,'31571073452189', 'ZX941818B'),('Steve','Rodgers',20000,'52818372454532','BG165344K'),
+    ('Clark','Kent', 45000,'27701114874367','XS633662U'),('Kim-Jong','Un', 10000, '28899322343434','FD714955B'),
+    ('Obiwon','Kenobi',400000,'74044710564354','YU617277W');
+
+Select*from Delivery_Employee;
```
Comment (jkelso662) [inline Delivery_Employee_table.sql]: This doesn't look the best! Capitalisation and also space needed either side of *

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Everything in your procedure is not indented enough! But requesting you at least fix the `Select*from` mess line 26. Merci buckets!

**Pull request merged**